lib/internals/src/utils/action/action-name-creator.ts

```diff
@@ -25,5 +25,5 @@ export function actionNameCreator(options: ActionNameCreatorOptions): string {
         }
     }
 
-    return `@${stateName}.${methodName}(${argsList})`;
+    return `@${statePath.split('.').join('/')}.${methodName}(${argsList})`;
```

Comment:
what do you think about 

```ts
statePath.replace(/\./g, '/')
```

instead 

```ts
statePath.split('.').join('/')
```

Reply:
sure, if that is more performant 👍

Reply:
investigate permormance?

Reply:
Seems like your intuition was right. Let's go with replace: 
https://stackoverflow.com/a/50463973/341837